File: labs/lab1/post/src/main/java/edu/miu/post/domain/Post.java
package edu.miu.post.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Post {
    private long id;
    private String title;
    private String content;
    private String author;
}

File: labs/lab8/src/components/Post.js
import React from 'react';
import "./post.css";

const Post = (props) => {
    return ( 
        <div className="content">
             <span>Id: </span>{props.id} <br/>
             <span>Title: </span>{props.title} <br/>
             <span>Author: </span>{props.author} <br/>
         </div>
     );
}
 
export default Post;
